esx_service/vmdk_ops.py

```diff
@@ -166,7 +166,9 @@ def createVMDK(vmdk_path, vm_name, vol_name, opts={}, vm_uuid=None, tenant_uuid=
     logging.info("*** createVMDK: %s opts = %s", vmdk_path, opts)
 
     if os.path.isfile(vmdk_path):
-        return err("File %s already exists" % vmdk_path)
+        # We are mostly here due to race or Plugin VMCI retry #1076
+        logging.warning("File %s already exists", vmdk_path)
+        return None
```

Comment:
Lets at least check that the volume opts in the KV match whats provided. Two create requests with say different VM sizes should be caught, unless the two volumes would be having the exact same properties.